packages/cubejs-schema-compiler/test/integration/mssql/MSSqlDbRunner.js

```diff
@@ -98,18 +109,20 @@ export class MSSqlDbRunner extends BaseDbRunner {
         MSSQL_PID: 'Developer',
         MSSQL_SA_PASSWORD: this.password(),
       })
+      .withExposedPorts(this.port())
       .withHealthCheck({
         test: [
           'CMD-SHELL',
-          `/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${this.password()} -Q "SELECT 1" || exit 1`
+          `${this.sqlcmdPrefix(version)}sqlcmd -C -l 1 -S localhost -U sa -P ${this.password()} -Q "SELECT 1" || exit 1`
         ],
-        interval: 2 * 1000,
-        timeout: 3 * 1000,
-        retries: 5,
-        startPeriod: 10 * 1000,
+        interval: 1000,
+        timeout: 1100,
+        retries: 20,
+        startPeriod: 2 * 1000,
       })
-      .withExposedPorts(this.port())
       .withWaitStrategy(Wait.forHealthCheck())
+      // .withWaitStrategy(Wait.forLogMessage('SQL Server is now ready for client connections'))
```

Comment:
Maybe, we don't want this commented line.

Reply:
fixed!